3-platform/order/spec/src/main/java/com/_6core/platform/orderspec/rest/v1/OrderApi.java

```diff
@@ -0,0 +1,60 @@
+package com._6core.platform.orderspec.rest.v1;
+
+import com._6core.platform.orderspec.rest.v1.dto.OrderResponse;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.Parameter;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.responses.ApiResponses;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestHeader;
+import reactor.core.publisher.Flux;
+
+@Tag(name = "Order Specification", description = "REST reactive specification for interaction with Order service")
+public interface OrderApi {
+    // Dummy property right now. Should be bringing to .property?
+    final String version = "6core.com.orders.v1";
+
+    /**
+     * Searching orders history by single user GET /orders
+     * 
+     * @param acceptHeader - (while request should use application version)
+     * @return OK (status code 200)
+     */
+    @GetMapping(value = "/orders", produces = "application/" + version + "+json")
```

Comment:
lets use application/json for sake of simplicity
```suggestion
    @GetMapping(value = "/orders", produces = "application/json")
```